flowzone.yml

```diff
@@ -793,19 +770,40 @@ jobs:
           cmd: bash -c "echo $INPUT | tr -d '[:space:]'"
           separator: ","
 
+      - name: Check for Docker compose files
+        id: docker_compose_files
+        uses: kanga333/json-array-builder@c7cd9d3a8b17cd368e9c2210bc3c16b0e2714ce5 # v0.2.1
+        with:
+          cmd: bash -c "find -L ${{ github.workspace }}/${{ inputs.working_directory }} -maxdepth 1 -type f -name 'docker-compose*.yml' -o -name 'docker-compose*.yaml'" || true
+          separator: newline
+
+      # only if docker-compose test files are found
+      # merge all compose files in the project and record the merged config as a multiline yaml output
+      - name: Pre-process Docker compose files
```

Comment:
@klutchell this steps breaks since it's missing .env file ([e.g.](https://github.com/balena-io/environment-staging/actions/runs/4001511798/jobs/6867816155#step:39:14))

Do we need to move/copy .env generation from the docker steps below up here or would `touch .env` suffice?

Reply:
I think I can just write our generate compose to the same working directory as the other compose files and we will be good. Let me open a PR and change the tests.